src/nyc_trees/apps/census_admin/tasks.py

```diff
@@ -27,6 +29,15 @@
 from apps.survey.models import Blockface
 from apps.event.models import Event
 
+_DATETIME_FIELDS = ('created_at',
```

Comment:
This list omits some fields (``last_login``, ``date_joined``, maybe more). Is walking the model metadata to find datetime fields an option?